flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/restart/NoOrFixedIfCheckpointingEnabledRestartStrategyFactory.java

```diff
@@ -23,7 +23,7 @@
  * depending if checkpointing was enabled.
  */
 public class NoOrFixedIfCheckpointingEnabledRestartStrategyFactory extends RestartStrategyFactory {
-	private static final long DEFAULT_RESTART_DELAY = 0;
+	private static final long DEFAULT_RESTART_DELAY = 10000L;
```

Comment:
Maybe use configuration here is better

Reply:
@klion26 thanks for review, this is default value, no need use configuration as [restart_strategies](https://ci.apache.org/projects/flink/flink-docs-release-1.7/dev/restart_strategies.html)  doc says, it's value is 10000L.